modules/ROOT/pages/alternatives.adoc

```diff
@@ -1,37 +1,21 @@
 = Setting alternatives
 
-Due to an https://github.com/fedora-sysv/chkconfig/issues/9[ongoing issue] in how alternatives configurations are stored on the system, Fedora CoreOS systems can not use the usual `alternatives` commands to configure them.
+Now Fedora CoreOS systems can use the usual `alternatives` commands to configure the default command, as https://github.com/fedora-sysv/chkconfig/issues/9[issue] is fixed.
```

Comment:
Let's move the fixed issue reference to the bottom or completely remove it as we don't reference all fixed issues in the doc.